napari/utils/transforms/transforms.py

```diff
@@ -272,6 +293,11 @@ def expand_dims(self, axes: Sequence[int]) -> 'ScaleTranslate':
         translate[not_axes] = self.translate
         return ScaleTranslate(scale, translate, name=self.name)
 
+    @cached_property
```

Comment:
```suggestion
    @property
```

Nit: this always returns true, so just a plain property seems fine.